### ISSUE TITLE
🦠 Model Request: Prediction of Aqueous Kinetic Solubility

### ISSUE THREAD

**Issue by pauline-banye** [ersilia-os/ersilia#509]
### Model Name

Aqueous Kinetic Solubility

### Model Description

This assay was used for kinetic solubility determination. Prediction of Aqueous solubility is one of the most important properties in drug discovery, as it has profound impact on various drug properties, including biological activity, pharmacokinetics (PK), toxicity, and in vivo efficacy. 

### Slug

aqueous-kinetic-solubility

### Tags

admemodel, kineticsolubility, prediction, solubility

### Publication

[Predictive models of aqueous solubility of organic compounds built on A large dataset of high integrity](https://pubmed.ncbi.nlm.nih.gov/31176566/)

### Code

https://github.com/ncats/ncats-adme

**Comment by GemmaTuron** [comment]
I am adding this here to help in the model results interpretation. This model should give two probabilities, one for solubility (>10 ug/mL) and the other for High solubility:
 _Compounds with solubility lower than 10 μg/mL are usually classified as insoluble. Selection of this classification criterion resulted in a balanced dataset with 51.1% soluble compounds and is favorable for QSAR modeling. On the other hand,
a minimum solubility of 52 μg/mL was suggested for a compound with an average permeability to be completely absorbed.4 Therefore, a second classification threshold of 50 μg/mL was adopted in this study to separate compounds with ideal solubility for drug development_

I'd try to give the output of both in a single file @pauline-banye

**Comment by GemmaTuron** [comment]
Hi @pauline-banye 

Please can you summarize what is the issue you are encountering when running the model through the Ersilia CLI? you mentioned the output was incorrect.
Also, have you been able to check that the output might be two values?
Thanks!

**Comment by GemmaTuron** [comment]
@pauline-banye , see my coments above about the double output of this model. we need to make sure which one is the one we are passing to the end users.
Have you compared results obtained with their code vs what is obtained by the model we have implemented?

**Comment by GemmaTuron** [comment]
This has been continued in #589
